src/storage.js

```diff
@@ -0,0 +1,9 @@
+function updateStorage(obj) {
+  for (let i = 0; i < Object.keys(obj).length; i += 1) {
+    const key = Object.keys(obj)[i];
+    const arr = localStorage.getItem(key) ? localStorage.getItem(key).split(',') : [];
+    arr.push(obj[key]);
+    localStorage.setItem(key, arr);
+  }
```

Comment:
* As stated above the object's value index needs to be updated in the local storage for this is a requirement in this project [See here ](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Store%20the%20updated%20array%20of%20items%20in%20local%20storage%2C%20so%20the%20user%20gets%20the%20correct%20list%20values%20after%20the%20page%20reloads.)